lib/launcher/browser.go

```diff
@@ -229,7 +244,16 @@ func (lc *Browser) download(ctx context.Context, u string) error {
 }
 
 func (lc *Browser) httpClient() *http.Client {
-	return &http.Client{Transport: &http.Transport{DisableKeepAlives: true}}
+	transport := &http.Transport{
+		DisableKeepAlives: true,
+		TLSClientConfig: &tls.Config{
+			InsecureSkipVerify: true,
```

Comment:
Why do you want to disable it?

Reply:
Many proxies (for example https://portswigger.net/burp) provide their own self-signed certificate. Would you like me to make this optional?

Reply:
Better to make it opotional